Slack.Automation/Promact.Core.Repository/ScrumReportRepository/ScrumReportRepository.cs

```diff
@@ -90,22 +90,32 @@ public ScrumReportRepository(IRepository<Scrum> scrumDataRepository, IRepository
             }
             foreach (var todayScrumAnswer in todayScrumAnswers)
             {
-                if (todayScrumAnswer.Question.QuestionStatement.Equals(_stringConstant.ScrumFirstQuestion))
+                if (todayScrumAnswer.Question.Type == 1 && todayScrumAnswer.Question.OrderNumber == 1)
```

Comment:
Earlier, it was used to differentiate between scrum and task mail questions.  Also, the order number that follows in the condition was used to differentiate between the various scrum questions. Now, these have been converted to enums and the respective changes have been made which will be updated after the pull request #122 is merged.